content/introduction/_index.md

```diff
@@ -1,60 +1,188 @@
 ---
-title: Introduction
+title: "Introduction"
 weight: 1
 pre: '<i class="fas fa-fw fa-comments"></i> '
 ---
 
-Welcome to the libp2p documentation portal! Whether you’re just learning how to build peer-to-peer systems with libp2p, want to dive into peer-to-peer concepts and solutions, or are looking for detailed reference information, this is the place to start.
+Welcome to the official libp2p documentation hub!
 
-## Overview
+Whether you’re just starting to dive into peer-to-peer concepts and 
+solutions, learning how to build peer-to-peer systems with libp2p, or 
+are looking for detailed reference information, this is the place to 
+start.
 
-Head over to [What is libp2p?](/introduction/what-is-libp2p/) for an introduction to the basics of libp2p and an overview of the problems it addresses.
+## What is libp2p?
 
+libp2p is a modular system of *protocols*, *specifications* and *libraries* 
+that enable the development of peer-to-peer network applications. 
 
-## Tutorials
+**The networking protocol of web3**
```

Comment:
```suggestion
**A p2p networking stack**
```

libp2p is also used outside of the blockchain space, and I'd like our branding to reflect that we're (aspiring to be a) general-purpose p2p networking stack.